src/open_inwoner/react/components/Accordion/Accordion.scss

```
@@ -0,0 +1,44 @@
+// TODO create design tokens.
+:root {
+  --oip-accordion-margin-block-start: 40px;
+  --oip-accordion-margin-block-end: 0;
+
+  --oip-accordion-summary-border-bottom-width: 1px;
+  --oip-accordion-summary-border-bottom-style: solid;
+  --oip-accordion-summary-border-bottom-color: var(--color-gray);
+  --oip-accordion-summary-margin-block-end: 16px;
+  --oip-accordion-summary-padding-block-start: 0;
+  --oip-accordion-summary-padding-block-end: 8px;
```

(no replacing) but add more margin values

```suggestion
      --oip-accordion-heading-margin-block-start: 0;
      --oip-accordion-heading-margin-block-end: 0;
      --oip-accordion-heading-margin-inline-start: 0;
      --oip-accordion-heading-margin-inline-end: 0;
```